core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -237,8 +264,8 @@ void killByKiller() throws InterruptedException {
          * @return The first non-null {@link VetoCause} provided by a process killing veto extension for this OSProcess. 
          * null if no one objects killing the process.
          */
-        protected @CheckForNull VetoCause getVeto() {
-            for (ProcessKillingVeto vetoExtension : ProcessKillingVeto.all()) {
+        protected @CheckForNull VetoCause getVeto() throws InterruptedException {
```

Comment:
According to GitHub, the only current usage appears to be within Jenkins itself.  Granted, someone could have used it in a private version, but I think that's unlikely.